src/itinerary/Itinerary.story.mdx

```diff
@@ -0,0 +1,331 @@
+            isoDate: '2017-12-11T09:00',
+            time: '09:00',
+            href: '#',
+          },
+          {
+            stepAriaLabel: 'Drop off location',
+            mainLabel: 'Bordeaux',
+            subLabel: (
+              <Proximity value={Distances.CLOSE} title="Distance from the drop off point" />
+            ),
+            isoDate: '2017-12-11T12:00',
+            time: '12:00',
+          },
+        ]}
+        headline="Mon 11 December"
+        highlightRoad
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## With stopovers
+
+<Preview>
+  <Story name="With stopovers">
+    <BaseSection>
+      <Itinerary
+        aria-label="Ride plan"
+        fromAddon="Lille"
+        fromAddonAriaLabel="Driver departure"
+        toAddon="Biarritz"
+        toAddonAriaLabel="Driver arrival"
+        places={placesWithStopovers}
+        headline="Mon 11 December"
+        highlightRoad
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## With segments & stopovers
+
+<Preview>
+  <Story name="With segments & stopovers">
+    <BaseSection>
+      <Itinerary
+        aria-label="Ride plan"
+        fromAddon="Lille"
+        fromAddonAriaLabel="Driver departure"
+        toAddon="Biarritz"
+        toAddonAriaLabel="Driver arrival"
+        places={[]}
+        segmentCollapsedLabels={['2 stops', '4 stops']}
+        segments={[places, places2]}
+        headline="Mon 11 December"
+        highlightRoad
+        isCollapsible
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## With single stopover
+
+<Preview>
+  <Story name="With single stopover">
+    <BaseSection>
+      <Itinerary
+        fromAddon="Lille"
+        toAddon="Biarritz"
+        places={[
+          {
+            time: '09:00',
+            isoDate: '2017-12-11T09:00',
+            subLabel: 'Porte de Vincennes',
+            mainLabel: 'Paris',
+          },
+        ]}
+        headline="Mon 11 December"
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## Without time
+
+<Preview>
+  <Story name="Without time">
+    <BaseSection>
+      <Itinerary
+        fromAddon="Lille"
+        toAddon="Biarritz"
+        places={[
+          {
+            isoDate: '2017-12-11T09:00',
+            subLabel: 'Porte de Vincennes',
+            mainLabel: 'Paris',
+          },
+          {
+            isoDate: '2017-12-11T15:00',
+            subLabel: 'Gare Bordeaux Saint-Jean',
+            mainLabel: 'Bordeaux',
+          },
+        ]}
+        headline="Mon 11 December"
+        highlightRoad
+      />
+    </BaseSection>
+  </Story>
+</Preview>
+
+## Usage
+
+```jsx
+import { Itinerary } from '@blablacar/ui-library/build/Itinerary'
+;<div>
+  <h1 id="titleId">Title example</h1>
+  <Itinerary
+    aria-labelledby="titleId"
+    fromAddon="Lille"
+    fromAddonAriaLabel="Driver departure"
+    toAddon="Biarritz"
+    toAddonAriaLabel="Driver arrival"
+    places={placesWithOneStopover}
```

Comment:
Can you include the `placesWithOneStopover` object in your example to demo its format?

Reply:
yes done